views/components/Sidebar/SideNavigation.css

```diff
@@ -0,0 +1,5 @@
+.SideNavigation_nav {
+  height: 664px;
```

Comment:
can we remove this height?

Reply:
This has to stay to make the space in between the bottom user info footer and the navigation

Reply:
I doubt this has to stay, we can almost definitely use margin/padding and not need explicit heights.

Reply:
[377af01](https://github.com/AMP-SCZ/dpdash/pull/458/commits/377af0123d164798059bf0dc152683f783a6b3f8)